src/vbk/pop_common.cpp

```diff
@@ -16,12 +16,6 @@ altintegration::PopContext& GetPop()
     return *app;
 }
 
-void StopPop()
-{
-    if (app == nullptr) return;
-    app->stop();
-}
-
```

Comment:
revert this change. 
call `shutdown` instead of stop

Reply:
FIxed